client/root.js

```diff
@@ -27,7 +27,7 @@ function setupLoggedOut() {
 	if ( config.isEnabled( 'desktop' ) ) {
 		page( '/', () => {
 			if ( config.isEnabled( 'oauth' ) ) {
```

Comment:
Another issue: `window.configData.oauth` is currently `undefined` in the desktop app. There is also no `oauth` key in `config/desktop.json` that overrides this ([link](https://github.com/Automattic/wp-calypso/blob/882a1bab18262b53340a0084e207cb0565f82c25/config/desktop.json#L2-L137)).